contracts/service-registry/src/msg.rs

```diff
@@ -79,3 +79,8 @@ pub enum QueryMsg {
         verifier: String,
     },
 }
+
+#[cw_serde]
+pub struct MigrateMsg {
+    pub coordinator_contract: Addr,
```

Comment:
We just renamed the field, so you don't need to pass this address in. You don't even need to define a message here, you can just use `Empty`.

Reply:
oh Ok, I was using that Empty but Edwin told me he used message for the previous migration so I changed it. Will revert back to not passing in the coordinator address and remove the message as well.